docs/BreakingChanges/002 XML schema validation is stricter.md

```diff
@@ -0,0 +1,22 @@
+## 2: XML schema validation is stricter
+
+### Scope
+Minor
+
+### Version Introduced
+4.5
+
+### Change Description
+In the .NET Framework 4.5, XML schema validation is more strict. If you use xsd:anyURI to validate a URI such as a mailto protocol, validation fails if there are spaces in the URI. In previous versions of the .NET Framework, validation succeeded. The change affects only applications that target the .NET Framework 4.5.
```

Comment:
Nit: "In the .NET Framework 4.5" sounds odd, consider "In .NET Framework 4.5" :)